# request params
sensor_msgs/Image color_image
---
# response params
bool is_success